osbs/conf.py

```diff
@@ -292,7 +292,7 @@ def get_registry_api_versions(self, platform=None):
                                 self.conf_section,
                                 "registry_api_versions",
                                 default='v1,v2')
-        versions = value.split(',')
+        versions = [x.strip() for x in value.split(',')]
```

Comment:
This pattern is used in two more places in this module. Maybe we should fix those as well?
```
$ git grep -n "value\.split(',')"
conf.py:279:            return value.split(',')
conf.py:286:            return value.split(',')
conf.py:295:        versions = [x.strip() for x in value.split(',')]
conf.py:396:            return [x.strip() for x in value.split(',')]
conf.py:403:            return [x.strip() for x in value.split(',')]
conf.py:540:            return [x.strip() for x in value.split(',')]
```

Reply:
Good idea, replaced other places where `split(',')` results were not stripped